src/core/remote.js

```diff
@@ -767,6 +767,31 @@ Remote.prototype.request = function(request_) {
 };
 
 /**
+ *  Create options object from positional function arguments
+ *
+ * @param {Array} params function parameters
```

Comment:
I like this form: `@param {Array} params - function parameters`.  Using a `-` after the param name or whatever provides some separation.